.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "trailingComma": "none",
```

Comment:
It's not a problem, but trailingComma makes diffs cleaner

Reply:
I agree! Fixed in this commit
https://github.com/jsolly/GeoAsteroids/commit/828840a103078d50a3d1c16c5a78a4b2c757f7fc